Monika After Story/game/script-topics.rpy

```diff
@@ -12777,3 +12777,24 @@ label monika_sleigh:
     show monika 5hkbfa at t11 zorder MAS_MONIKA_Z with dissolve
     m 5hubfa "An experience like that with you would be so breathtaking~"
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_enjoyingspring",category=['spring'],prompt="Enjoying Spring",random=True))
```

Comment:
Shouldn't default random, but should default to if it's spring. Also, all words after the first one in a prompt aren't capitalized. Also, needs zz_seasons changes. Do you want me to add those for you?
```suggestion
    addEvent(Event(persistent.event_database,eventlabel="monika_enjoyingspring",category=['spring'],prompt="Enjoying spring",random=mas_isSpring()))
```